tracer/src/Datadog.Trace/AppSec/Transports/Http/HttpTransport.Framework.cs

```diff
@@ -67,17 +73,26 @@ public IHeadersCollection GetResponseHeaders()
 
         public void WriteBlockedResponse(string templateJson, string templateHtml)
         {
-            // todo
-        }
+            _context.Items["block"] = true;
+            var httpResponse = _context.Response;
+            httpResponse.Clear();
+            // httpResponse.Headers.Clear();
```

Comment:
Are the response headers handled somewhere else?

Reply:
thanks, yes, definitely shouldn't be commented. We can't remove the headers in all circumstances, but we can do better that that.

Fixed in e9b5f81f570178a129d60b598d9ec96619d2a755